app/views/analytics/facilities/show.html.erb

```diff
@@ -49,28 +62,55 @@
     <% end %>
   </p>
 
-  <% if policy([:cohort_report, @facility]).patient_list? %>
-    <p>
-      <%= link_to analytics_facility_patient_list_path(@facility) do %>
-        <i class="far fa-file-excel mr-2"></i>
-        Patient line list
-      <% end %>
-    </p>
-    <p>
-      <%= link_to analytics_facility_patient_list_with_history_path(@facility) do %>
-        <i class="far fa-file-excel mr-2"></i>
-        Patient line list (with medication history)
-      <% end %>
-    </p>
+  <% if Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) %>
+    <% if current_admin.accessible_facilities(:view_reports).include?(@facility) %>
```

Comment:
This should be `:view_pii`, right? For the line list?

Reply:
Missed this, switching it to `view_pii`